projects/angular/src/button/button-group/_button-group.clarity.scss

```diff
@@ -4,17 +4,20 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-@import '../../popover/dropdown/menu-mixins.clarity';
+@use '../../popover/dropdown/menu-mixins.clarity';
```

Comment:
I think `menu-mixins` is not used in this file.

Reply:
Removed in commit c398468.